tools/makemanifest.py

```diff
@@ -38,14 +38,22 @@ def include(manifest):
 
     The manifest argument can be a string (filename) or an iterable of
     strings.
+
+    Relative paths are resolved with respect to the current file.
```

Comment:
Is 'the current file' the `manifest` argument?

Reply:
It's relative to the current file being executed. i.e. the file that called `include`. So it's not relative to the `manifest` argument, rather to the current file.